src/industry_gui.cpp

```diff
@@ -392,6 +396,13 @@ class BuildIndustryWindow : public Window {
 		this->FinishInitNested(0);
 
 		this->SetButtons();
+
+		/* Show scenario editor tools in editor */
```

Comment:
sentences end with a dot. Please add a dot at the end of a sentence in a comment :)